js/snippetPreview.js

```diff
@@ -1146,6 +1153,52 @@ SnippetPreview.prototype.setMetaDescription = function( metaDesc ) {
 	this.changedInput();
 };
 
+/**
+ * Creates elements with the purpose to calculate the sizes of elements and puts these elemenents to the body.
+ */
+SnippetPreview.prototype.createMeasurementElements = function() {
+	var titleElement = document.createElement( 'span' );
+	var metaDescriptionElement = document.createElement( 'span' );
+
+	titleElement.id = 'measureTitle';
+	titleElement.style.width = 'auto';
```

Comment:
Yes, in this case it's a temporary hidden object. That's the reason I did it this way. @atimmer What do you think?